the en_AU date/time definitions are incomplete, needs to be fixed, see 23217745 
and 23558523 for details.

--- core.orig/common/main/en_AU.xml
+++ core/common/main/en_AU.xml
@@ -48,23 +48,21 @@
 				<months>
 					<monthContext type="format">
 						<monthWidth type="abbreviated">
-							<month type="1">Jan.</month>
-							<month type="2">Feb.</month>
-							<month type="3">Mar.</month>
-							<month type="4">Apr.</month>
-							<month type="6">Jun.</month>
-							<month type="7">Jul.</month>
-							<month type="8">Aug.</month>
-							<month type="9">Sep.</month>
-							<month type="10">Oct.</month>
-							<month type="11">Nov.</month>
-							<month type="12">Dec.</month>
+							<month type="1">Jan</month>
+							<month type="2">Feb</month>
+							<month type="3">Mar</month>
+							<month type="4">Apr</month>
+							<month type="5">May</month>
+							<month type="6">Jun</month>
+							<month type="7">Jul</month>
+							<month type="8">Aug</month>
+							<month type="9">Sep</month>
+							<month type="10">Oct</month>
+							<month type="11">Nov</month>
+							<month type="12">Dec</month>
 						</monthWidth>
 					</monthContext>
 					<monthContext type="stand-alone">
-						<monthWidth type="abbreviated">
-							<month type="5">May</month>
-						</monthWidth>
 						<monthWidth type="narrow">
 							<month type="1">Jan.</month>
 							<month type="2">Feb.</month>
@@ -167,12 +165,12 @@
 				<dateFormats>
 					<dateFormatLength type="long">
 						<dateFormat>
-							<pattern>dMMMM,y</pattern>
+							<pattern>d MMMM, y</pattern>
 						</dateFormat>
 					</dateFormatLength>
 					<dateFormatLength type="medium">
 						<dateFormat>
-							<pattern>dMMM,y</pattern>
+							<pattern>d MMM, y</pattern>
 						</dateFormat>
 					</dateFormatLength>
 					<dateFormatLength type="short">
